android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirCarePlanGenerator.kt

```diff
@@ -214,8 +214,11 @@ constructor(
 
         carePlan.contained.clear()
 
+        // TODO Confirm why reversion here
         // Save CarePlan only if it has activity, otherwise just save contained/dependent resources
-        if (output.hasActivity()) defaultRepository.addOrUpdate(true, carePlan)
+        // if (output.hasActivity()) defaultRepository.addOrUpdate(true, carePlan)
```

Comment:
what the context here?

Reply:
We can remove the comments, I confirmed with the team that it was an inadvertent a merge from main

Reply:
cool, I'll update...